### PR TITLE
Ng fab btns routing

### PR DESCRIPTION
**Pull request by Nikolay-Gavazov**

### CODE REVIEW

File: angular.json
Comment (Getsov) [inline angular.json:1]: What is this analytics and why is needed?

File: src/app/tabs/tabs.page.html [hunk 1]
```diff
@@ -1,20 +1,41 @@
...
-      <ion-label>Tab 2</ion-label>
+    <ion-tab-button tab="calendar" href="/calendar">
+      <ion-icon aria-hidden="true" name="calendar"></ion-icon>
+      <ion-label>Calendar</ion-label>
     </ion-tab-button>
 
-    <ion-tab-button tab="tab3" href="/tabs/tab3">
-      <ion-icon aria-hidden="true" name="square"></ion-icon>
-      <ion-label>Tab 3</ion-label>
+    <ion-tab-button id="open-modal" expand="block" href="javascript:void(0)">
```
Comment (Getsov) [inline src/app/tabs/tabs.page.html]: There shouldn’t be any javascript:void hrefs in the templates

File: src/app/tabs/tabs.page.html [hunk 2]
```diff
@@ -1,20 +1,41 @@
...
+      <ion-icon aria-hidden="true" name="search"></ion-icon>
+      <ion-label>Search</ion-label>
     </ion-tab-button>
+    <!-- <ion-button id="open-modal" expand="block">Search</ion-button> -->
+    
+    <ion-modal #modal trigger="open-modal" [initialBreakpoint]="0.25" [breakpoints]="[0, 0.25, 0.5, 0.75]">
```
Comment (Getsov) [inline src/app/tabs/tabs.page.html]: This shouldn’t be here, at minimum it should be as a separate component

**Comment by Nikolay-Gavazov** [comment]
I made all that we spoke about.

File: src/app/tabs/tabs.page.html [hunk 3]
```diff
@@ -1,20 +1,31 @@
...
+      <ion-icon aria-hidden="true" name="car-sport-outline"></ion-icon>
+      <ion-label>Events</ion-label>
     </ion-tab-button>
+
+    <ion-tab-button tab="notifications" href="/notifications">
+      <ion-icon aria-hidden="true" name="notifications"></ion-icon>
+      <ion-label>Notifications</ion-label>
+    </ion-tab-button>
+
+    <ion-tab-button tab="profile" href="/tabs/profile">
```
Comment (Getsov) [inline src/app/tabs/tabs.page.html]: is this working?

Comment (Nikolay-Gavazov) [reply src/app/tabs/tabs.page.html]: Which Line?

Comment (Getsov) [reply src/app/tabs/tabs.page.html]: The commented one - 24

Comment (Nikolay-Gavazov) [reply src/app/tabs/tabs.page.html]: Yes, i Check it. Now I will change it.

**Review by Getsov — APPROVED** [review]
Approved, but PR is a bit long now and harder to read and follow, try to be smaller ones next time

**Pull request merged**